### ISSUE TITLE
Remove trailing slash after gu (urlparent)

### ISSUE THREAD

**Issue by mabgnu** [tridactyl/tridactyl#1565]
Currently, pressing `gu` (which calls `urlparent`) leaves a trailing slash.  For instance

```
http://git.sr.ht/~bandali/dotfiles -> http://git.sr.ht/~bandali/
```

whereas I would like

```
http://git.sr.ht/~bandali/dotfiles -> http://git.sr.ht/~bandali
```

I’d like to request an option be added that would make `gu` remove the trailing slash when going to the parent url.

**Comment by rob-3** [comment]
This provides the desired behavior when urlparenttrailingslash is set to false.

**Comment by bovine3dom** [comment]
Closed in #1718. Thanks, @rob-3.
